test/integration/boolean_constraints/launch.sh

```diff
@@ -13,50 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#################################################################
-#   PLEASE FILL THE VARIABLES WITH VALID VALUES FOR TESTING     #
-#   DO NOT REMOVE/ADD ANY OF THE VARIABLES                      #
-#################################################################
-
-#################################################################
-#   RESOURCE IDENTIFIERS, VARIABLES AND ROLES                   #
-#################################################################
-
-# Project
-export PROJECT_ID="base-project-196723"
-
-# Organization
-export ORGANIZATION_ID="65779779009"
-
-# 1 above organization's folder to exclude from policy
-export FOLDER_EXCLUDE="886025180534"
-
-# Folder
-export FOLDER_ID="886025180534"
-
-# 1 above folder's project to exclude from policy
-export PROJECT_EXCLUDE="base-project-196723"
-
-
-# Constraints
-
-# Applied to: project
-export PROJECT_CONSTRAINT="constraints/compute.disableSerialPortAccess"
-
-# Applied to: folder
-export FOLDER_CONSTRAINT="constraints/compute.restrictXpnProjectLienRemoval"
-
-# Applied to: org
-export ORG_CONSTRAINT="constraints/compute.disableSerialPortAccess"
-
-# Credentials
-export CREDENTIALS_PATH="/vagrant/terraform5.json"
-export CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=$CREDENTIALS_PATH
-
 #################################################################
 #   FUNCTIONS FOR PREPARING WORKSPACE AND CALLING BATS          #
 #################################################################
 
+# shellcheck disable=SC1091
```

Comment:
is it possible to put this outside launch.sh like source ./sourse.sh && launch.sh?

Reply:
probably. all of that will go away anyway, the module will be migrated to kitchen tests